File: Aprendendo-Java/Exemplos/Exemplos/src/exemplos/matrizes.java
/*
 * Click nbfs://nbhost/SystemFileSystem/Templates/Licenses/license-default.txt to change this license
 * Click nbfs://nbhost/SystemFileSystem/Templates/Classes/Class.java to edit this template
 */
package exemplos;
import java.util.Scanner;

public class matrizes {
    public static void main(String[] args){
    
        Scanner sc = new Scanner(System.in);
        
        int m,n;
        
        System.out.print("Quantas linhas vai ter a matriz? ");
        m = sc.nextInt();
        System.out.print("Quantas colunas vai ter a matriz? ");
        n= sc.nextInt();
        
        int[][]mat = new int[m][n];
        
        for(int i =0; i <m;i++){
            for(int j = 0; j <n; j++){
                System.out.print("Elemento [" + i + "," + j + "]: ");
                mat[i][j] = sc.nextInt();
            }            
        }
        
        System.out.println("\nMATRIZ DIGITADA:");
        for (int i = 0; i < m; i++) {
            for (int j = 0; j < n; j++) {
                System.out.println(mat[i][j] + " ");
            }
            System.out.println();
        }
        
        sc.close();

    }
}

File: Aprendendo-Python/Exemplos/matrizes.py
m = int(input("Quantas linhas vai ter a matriz? "))
n = int(input("Quantas colunas vai ter a matriz "))

mat: [[int]] = [[0 for x in range(n)] for x in range(m)]

for i in range(0,m):
    for j in range(0,n):
        mat[i][j] = int(input(f"Elemento da [{i},{j}]: "))

for i in range(0,m):
    for j in range(0,n):
        print(f"{mat[i][j]}", end=" ")
    print(" ")
